std/stdio.d

```diff
@@ -4517,6 +4517,13 @@ Initialize with a message and an error code.
     }
 }
 
+enum: int
+{
+    stdin_handle  = 0,
+    stdout_handle = 1,
+    stderr_handle,
+}
+
 // Undocumented but public because the std* handles are aliasing it.
 @property ref File makeGlobal(int _iob)()
```

Comment:
Would it be bad to make the enum an actual type, and then make the template parameter an instance of that enum? This would prevent for instance `makeGlobal!5`, which then also aliases to `stderr` without something like that. At the very least, this should say `if(_iob >= stdin_handle && iob <= stderr_handle)`

Reply:
Yes, I'll do that instead. My goal was to be as unintrusive as possible.